Version.props

```diff
@@ -1,14 +1,18 @@
 <Project ToolsVersion="15.0">
-  <PropertyGroup Condition=" '$(PackageVersion)' == '' ">
-    <PackageReleaseVersion>1.0.1</PackageReleaseVersion>
-    <PackageBuildQuality>$(BUILD_QUALITY)</PackageBuildQuality>
-    <PackageBuildQuality Condition=" '$(PackageBuildQuality)' == '' ">beta1</PackageBuildQuality>
-    <PackageVersion>$(PackageReleaseVersion)-$(PackageBuildQuality)</PackageVersion>
+  <PropertyGroup Condition=" '$(PackageVersion)' == '' ">	
+    <!-- If $(PB_VersionStamp) is empty and $(PB_IsStable) is not true, then a default PackageBuildQuality will be added. -->
+    <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">1.0.1</PackageReleaseVersion>
+    <PackageBuildQuality>$(PB_VersionStamp)</PackageBuildQuality>
```

Comment:
Won't this rename break builds that are invoked via setup.cmd or setup.sh?

Reply:
These properties were only used from within this propertygroup, so shouldn't have any impact on the callers.

Reply:
setup.cmd and setup.sh passed the original property names though

Reply:
The only property name that I changed is BUILD_NUMBER and I couldn't see this getting set anywhere else.  PackageBuildQuality propertyname is still the same. This has not changed. Is this the point of concern?